examples/island/Island.cpp

```diff
@@ -156,8 +156,7 @@ int main()
     }
 
     // Center the status text
-    statusText.setPosition({(windowWidth - statusText.getLocalBounds().width) / 2.f,
-                            (windowHeight - statusText.getLocalBounds().height) / 2.f});
+    statusText.setPosition((sf::Vector2f(windowWidth, windowHeight) - statusText.getLocalBounds().size) / 2.f);
```

Comment:
Might be worthwhile changing `windowWidth` and `windowHeight` into a `windowSize` vector as well.

Reply:
I see more possible simplifications (https://github.com/kimci86/SFML/compare/feature/rect_as_two_vectors...kimci86:SFML:feature/simplify_island_example) that I would rather not add to this PR.